src/target/riscv/riscv-013.c

```diff
@@ -2811,6 +2815,162 @@ void riscv013_fill_dmi_nop_u64(struct target *target, char *buf)
 	buf_set_u64((unsigned char *)buf, DTM_DMI_ADDRESS_OFFSET, info->abits, 0);
 }
 
+static int riscv013_test_sba_config_reg(struct target *target, uint32_t address)
```

Comment:
Why is this limited to 32-bit addresses?

Reply:
Don't call this variable "address." Something like "bad_address" is much better. Having both addr and address in the same function is pretty confusing.

Reply:
Fixed in most recent version